### ISSUE TITLE
When a node is moved between two trees, move_node.jstree event is not triggered.

### ISSUE THREAD

**Issue by kelvinkawaiho** [vakata/jstree#2571]
Problems:

1) When a node is moved between two trees, move_node.jstree event is not triggered.  However, if I move the node within the same tree, the move_node.jstree is triggered.
2) After the node is moved between two trees, not all the content of the node is moved, e.g. ID in the new tree is changed (I can understand this part because the target tree may already have this ID already). However, I have "data" : { "abc" : "edf" }, the data node from the source tree is also not copied over to the target tree.

Basically, I can only see the "text" node moved over.

Thanks for taking a look.
